test/conformance/queue/urQueueCreate.cpp

```diff
@@ -12,6 +12,22 @@ TEST_P(urQueueCreateTest, Success) {
     ASSERT_SUCCESS(urQueueRelease(queue));
 }
 
+TEST_P(urQueueCreateTest, SuccessWithProperties) {
```

Comment:
Would it make sense to check the other valid flags as well using a parameterized test?